### ISSUE TITLE
Test content category labels (based on research content modeling sessions output)

### ISSUE THREAD

**Issue by yowill** [18F/dos-ops-center#136]
As an 18F researcher, I want to test proposed content category labels so that I can assess if the labels make sense for users and determine if they will be able to easily find the content they need.

AC:
- [ ] Create the content category labels (the working labels are Administrative Information, Published Ops Materials, Internal Ops Information and Reference Materials, Task Force, SCORE, and Interagency Papers and Projects). 
- [ ] Finalize the list of products to test with the category labels based on the list in #133 and #123 .
- [ ] Execute the tests.
- [ ] Summarize the results into a recommendation for category labels.

Support:
@vishaliyer-18f 
@matt-dobson 
@RyanSibley 
@jeanninehunter 
@MeghanLuecke

**Comment by RyanSibley** [comment]
Notes: https://docs.google.com/document/d/1mTQjOADqCYGocyhXcylpzVo2hDTeAQ_Q43u5md0EBdc/edit#

and 

https://docs.google.com/spreadsheets/d/1mFvUdWxCzK_aMavUhJyEkjgMgIVcDFBXt0yuouOYa7Q/edit#gid=489212529

and 

![image from ios](https://user-images.githubusercontent.com/1151139/53592100-8d4dd480-3b63-11e9-8887-0e7bef81a275.jpg)

also see notes and conversation in #133

**Comment by yowill** [comment]
@durland18F @MeghanLuecke Can we talk this through here?

@durland18F Can you tell @MeghanLuecke what she needs to give you to help you move forward? I think there has been some wires crossed here.

**Comment by durland18F** [comment]
@MeghanLuecke So, background is that we are testing the category labels Ops generated at the same time as the metadata we have also run by you (geographical, bureau, crisis). You and I talked via DM a little while back about getting some retrieval scenarios down from you and another person or two, and you indicated that you would like to wait for some more journey mapping to get these. Is that still true? Do you think today's ExecSec exercise will get us there? We also need to have some more subject line examples to create somewhat plausible "results." for this test, and I think we can give a more targeted request for this data once we have the scenarios.

**Comment by MeghanLuecke** [comment]
@durland18F thanks - this makes much more sense! I believe we are NOT ready to complete Issue 136 until we have shared at least the basics on CORE and General Alerts with you. Any test we create based solely on SOS and ExecSec Briefs will be incomplete. We can coordinate with Will and Kara on this, because I know we have concerns about overall sprint bandwidth, but Issue 136 should not come before the CORE and General Alert discussion. Thanks!

**Comment by yowill** [comment]
Ok, we've chatted with both sides here and are moving this to the backlog in place of #180 and #181 so that information can be included in this for a future Sprint.

**Comment by karareinsel** [comment]
@durland18F I thought this issue was different from #176 but based on what you're describing they sound like the same test. Can you double check the acceptance criteria above against what's in #176? I just want to clarify if we're talking about two distinct tests or if they've merged into one test. Thank you!

**Comment by karareinsel** [comment]
Adding the blocked label since it sounds like this issue is blocked by #180 and #181 .

**Comment by karareinsel** [comment]
Closing out this issue because it's duplicative with #176
